examples/jsm/loaders/IFCLoader.js

```diff
@@ -2,1131 +2,2069 @@ import {
 	IFCRELAGGREGATES,
 	IFCRELCONTAINEDINSPATIALSTRUCTURE,
 	IFCRELDEFINESBYPROPERTIES,
+	IFCRELASSOCIATESMATERIAL,
 	IFCRELDEFINESBYTYPE,
 	IFCPROJECT,
 	IfcAPI,
-} from './ifc/web-ifc-api.js';
+} from "./ifc/web-ifc-api.js";
 import {
 	BufferAttribute,
-	BufferGeometry,
 	Mesh,
 	Matrix4,
+	BufferGeometry,
 	Color,
 	MeshLambertMaterial,
 	DoubleSide,
-	Group,
 	Loader,
 	FileLoader,
-} from '../../../build/three.module.js';
-import * as BufferGeometryUtils from '../utils/BufferGeometryUtils.js';
-
-const IdAttrName = 'expressID';
-const merge = ( geoms, createGroups = false ) => {
-
-	return BufferGeometryUtils.mergeBufferGeometries( geoms, createGroups );
+} from "../../../build/three.module.js";
```

Comment:
@agviegas could you configure your eslint to use the repo styleguide?

Otherwise just run `npm run lint-fix` before the commit.

Reply:
Right, I'll do that as soon as I get back to my computer (17th).